cli/src/lib/paths.js

```diff
@@ -43,12 +47,26 @@ module.exports = (cwd = process.cwd()) => {
         shellAppDirname,
         shellApp: path.join(base, `./.d2/shell/${shellAppDirname}`),
         shellPublic: path.join(base, './.d2/shell/public'),
+        shellPublicManifestWebapp: path.join(
+            base,
+            './.d2/shell/public/manifest.webapp'
+        ),
+        shellPublicManifestJson: path.join(
+            base,
+            './.d2/shell/public/manifest.json'
+        ),
+        shellPublicConfigJson: path.join(
+            base,
+            './.d2/shell/public/d2.config.json'
+        ),
         shellBuildOutput: path.join(base, './.d2/shell/build'),
+        shellBuildServiceWorker: path.join(
+            base,
+            './.d2/shell/build/service-worker.js'
+        ),
 
         buildOutput: path.join(base, './build'),
         buildAppOutput: path.join(base, './build/app'),
-        buildAppManifest: path.join(base, './build/app/manifest.webapp'),
-        buildAppConfigJson: path.join(base, './build/app/d2.config.json'),
```

Comment:
These files are moved to the `shell/public/` dir (see above lines) so that they can be processed by `workbox-build` and added to the service worker precache manifest